shacl12-profiling/index.html

```diff
@@ -352,7 +352,10 @@
 
 		<section id="introduction">
 			<h2>Introduction</h2>
-			<p>SHACL Profiling is the act of creating <a href="https://www.w3.org/TR/dx-prof/#dfn-profile">profile</a> of <a href="https://www.w3.org/TR/shacl/#dfn-rdf-graph">RDF graphs</a> using SHACL...
+			<p>SHACL Profiling is the act of creating
+			a <a href="https://www.w3.org/TR/dx-prof/#dfn-profile">profile</a>
+			of a <a href="https://www.w3.org/TR/shacl/#dfn-rdf-graph">RDF graphs</a>
```

Comment:
Sorry, missed this earlier.
```suggestion
			of an <a href="https://www.w3.org/TR/shacl/#dfn-rdf-graph">RDF graph</a>
```